From 37344b708cf49feeda1992bb1da1e9bb2c68cfc9 Mon Sep 17 00:00:00 2001
From: Ian Denhardt <ian@zenhack.net>
Date: Wed, 6 Oct 2021 21:41:32 -0400
Subject: [PATCH 3/4] Hide 'new' buttons.

...since we intend this to be used in single-doc mode; this could cause
confusion.
---
 public/views/hedgedoc/header.ejs | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/public/views/hedgedoc/header.ejs b/public/views/hedgedoc/header.ejs
index 48565814e..76571face 100644
--- a/public/views/hedgedoc/header.ejs
+++ b/public/views/hedgedoc/header.ejs
@@ -22,8 +22,10 @@
                 <i class="fa fa-caret-down"></i>
             </a>
             <ul class="dropdown-menu list" role="menu" aria-labelledby="menu">
+<!--
                 <li role="presentation"><a role="menuitem" class="ui-new" tabindex="-1" href="<%- serverURL %>/new" target="_blank" rel="noopener"><i class="fa fa-plus fa-fw"></i> <%= __('New') %></a>
                 </li>
+-->
                 <li role="presentation"><a role="menuitem" class="ui-publish" tabindex="-1" href="#" target="_blank" rel="noopener"><i class="fa fa-share-square-o fa-fw"></i> <%= __('Publish') %></a>
                 </li>
                 <li class="divider"></li>
@@ -112,11 +114,13 @@
             </li>
         </ul>
         <ul class="nav navbar-nav navbar-right" style="padding:0;">
+<!--
             <li>
                 <a href="<%- serverURL %>/new" target="_blank" rel="noopener" class="ui-new">
                     <i class="fa fa-plus"></i> <%= __('New') %>
                 </a>
             </li>
+-->
             <li>
                 <a href="#" target="_blank" rel="noopener" class="ui-publish">
                     <i class="fa fa-share-square-o"></i> <%= __('Publish') %>
-- 
2.39.2

